lib/board.rb

```diff
@@ -17,7 +17,7 @@ def empty_board
   end
 
   def grid_format
-    printed_board = empty_board.map do |row|
+    printed_board = @grid.map do |row|
```

Comment:
empty_board.map changed to @grid.map in the method grid_format. Because empty_board was giving us the original board after we placed the "X". By changing to @grid, it allows the board to display the piece that was placed by the user.